### ISSUE TITLE
[0.9.1.8 beta release]Civics: Need a civic check on opening / destroy garbage bags

### ISSUE THREAD

**Issue by IcedForge** [StrangeLoopGames/EcoIssues#19638]
There is no way to reward picking up trash in the wild as there is no way for the civic system to check for picking up trash, only dropping it.

Red section is dropping the garbage
Green section is when i picked it up and destroyed the garbage bag.
![Screenshot_6.jpg](https://images.zenhubusercontent.com/5f5a7fa59878ed957a861886/05f7432f-e90f-41b9-9a19-5bec65e8aa46)

**Comment by WeaselDog** [comment]
There is a trigger for pick up / drop garbage block. I think you are looking for a trigger related to picking up trash item. I can see the confusion because it says "Drop Garbage" in the trigger.

**Comment by IcedForge** [comment]
Yes that is correct, this is for opening the actual bag and emptying it, clearing the debris not the actual "garbage block itself"
